Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +99,65 @@ namespace Terrain
                         ->Attribute(AZ::Edit::Attributes::ReadOnly, &MeshConfiguration::IsClodDisabled)
                     ;
 
+                editContext->Class<ClipmapConfiguration>("Clipmap", "Settings related to clipmap rendering")
+                    ->DataElement(AZ::Edit::UIHandlers::ComboBox, &ClipmapConfiguration::m_clipmapSize, "Clipmap image size", "The size of the clipmap image in each layer.")
+                        ->EnumAttribute(ClipmapConfiguration::ClipmapSize1024, "1024")
+                        ->EnumAttribute(ClipmapConfiguration::ClipmapSize512, "512")
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxRenderRadius, "Macro clipmap max render radius", "Max render radius that the lowest resolution clipmap can cover.")
+                        ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 1000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2000.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 5000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4000.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxRenderRadius, "Detail clipmap max render radius", "Max render radius that the lowest resolution clipmap can cover.")
+                        ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 50.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 100.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 2000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 1000.0f)
```

Comment:
In what way is this different from the "Detail Material Render Distance"? Does it make sense to expose both?

Reply:
"Detail Material Render Distance" was the old name of this `m_detailClipmapMaxRenderRadius`. Because we are only seeing the front, radius makes more sense imo.